tests/Type/WebMozartAssert/data/data.php

```diff
@@ -132,13 +132,21 @@ public function doFoo($a, $b, array $c, iterable $d, $e, $f, $g, $h, $i, $j, $k,
 
 		Assert::classExists($ag);
 		$ag;
+
+		/** @var object $bar */
+		$bar = new Bar();
+		Assert::methodExists($bar, 'foo');
+		$bar->foo();
```

Comment:
1) Remove the `/** @var */`
2) Instead of `$bar->foo()`, put just `$bar;` here and update the test that fails. The correct type here should be `Bar&hasMethod(foo)`.